frontend/app/helpers/get-level.js

```diff
@@ -0,0 +1,15 @@
+import Helper from "@ember/component/helper";
+import { inject as service } from "@ember/service";
+
+export default Helper.extend({
+  store: service(),
+
+  compute(level) {
+    return this.get("store")
+      .findAll("personRoleLevel")
```

Comment:
this triggers the person_role_levels api call a second time after loading all candidates for the drop down.
rename helper to: selectedPersonRoleLevel
can't you provide the list of levels as argument here?